### ISSUE TITLE
Text Stops Working....sometimes.

### ISSUE THREAD

**Issue by JthunderDraws** [JimmyAppelt/Snaz#133]
I originally posted this over on OBS:

Hello, I love the program, but I do have an issue. Minor issue, but when playing music, the text display will sometimes not work. I usually have to either restart snaz or pause, then play to fix. Not sure if bug, or just my comp being....my comp. Any feedback would be awesome.

I'm not sure what additional details I need to provide.  I use OBS 17.0.2, on Windows 7 Home Premium (64bit).  Pretty decent build, got an i5 (Devils Canyon), 750Ti SuperClocked, 16gb of ddr3 ram.  The program runs off of my C:/ drive, I usually use it while streaming to Picarto, while Itunes, chrome, and manga studio 5 are running at the same time.  Also it defaults to basic text I put up when I'm not playing music (no music=on break).  Most of the time I just pause then play, and it fixes it.   Can't tell if it happens at a specific time or not, it's happened while my cpu runs at 18-25% load, or between 45-65% load. 

I hope that's enough info, please let me know if I need to provide more.  Any feedback appreciated.  Thanks.

JT

**Comment by JimmyAppelt** [comment]
I assume you having the issue with ITunes ?

What version of Snaz do you have running ?

**Comment by JimmyAppelt** [comment]
And witch version of iTunes are you running ?

**Comment by JthunderDraws** [comment]
I use ITunes, yes.  

Snaz:  1.12.5.0
ITunes: 12.5.5.5

**Comment by FatherlyArson** [comment]
i also have this same problem 
and when i switch playlists in itunes
or select the seperate files option in snaz it crashes

Snaz: 1.12.5.0
ITunes: 12.5.5.5

**Comment by JimmyAppelt** [comment]
As far as for now, I seem to be unable to replicate this kind of behavior.

The separate files option, works as expected, and I would not see the connection between both (In part of crashing)

**Comment by JthunderDraws** [comment]
It did it again while I was streaming, so I made a recording.  Not sure if it will help, but here it is anyways:

https://youtu.be/9trESKiYCv0

OBS: 17.0.2 (windows, 64bit)
Snaz:  1.12.5.0
itunes: 12.5.5.5

**Comment by FatherlyArson** [comment]
it seems like there is some limit to how many songs get played before the info stops showing
i play 1 song on constant repeat and it eventually stops showing album image & text info but when i play
a playlist of about 5-6 songs, it takes longer but still happens.i think with a playlist it takes longer to happen because different songs are being played
snaz might have a problem refreshing the exact same info over time(thats's my asumption)
hope this helps
i tried this on a fresh install of snaz(without touching any settings).
OBS: 17.0.2 (windows, 64bit)
Snaz: 1.12.5.0
itunes: 12.5.5.5
this is still the best streamer app ive ever used

**Comment by JthunderDraws** [comment]
Of course!  Still use it, just a bug that I thought the developer would like to know about.  Thanks mr. Arson, I'm glad you had the patience to figure this out, appreciate it.
